### PR TITLE
[Docs] New file listing all the downloadable files during installation

### PR DESCRIPTION
**Pull request by rajkaramchedu**
A new doc listing all the downloadable files, like yaml, genesis and waypoint. Needed for either a validator node or a validator/public fullnode, so keeping it at a higher level. **Merge only when we finalize the haproxy files.** as per this [Slack thread.](https://aptos-org.slack.com/archives/C042WTC9S3A/p1665013166144089)

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4812)

### CODE REVIEW

**Review by teenu4 — CHANGES_REQUESTED** [review]
Hi @rajkaramchedu, we need to specify filename after -O param, for example:
`wget -O docker-compose.yaml https://raw.githubusercontent.com/aptos-labs/aptos-core/mainnet/docker/compose/aptos-node/docker-compose.yaml`

**Comment by rajkaramchedu** [comment]
> Hi @rajkaramchedu, we need to specify filename after -O param, for example: `wget -O docker-compose.yaml https://raw.githubusercontent.com/aptos-labs/aptos-core/mainnet/docker/compose/aptos-node/docker-compose.yaml`

I guess I was sleeping. Good catch!

**Review by teenu4 — APPROVED** [review]

File: developer-docs-site/docs/nodes/node-files.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+- **Git branch:** `mainnet` on https://github.com/aptos-labs/aptos-core
+- **Command to download:**
+    ```bash
+    wget -O docker-compose.yaml https://raw.githubusercontent.com/aptos-labs/aptos-core/mainnet/docker/compose/aptos-node/docker-compose.yaml
+    ```
+
+## validator.yaml
+
+- **Git repo:** `aptos-core`
+- **Git branch:** `main` on https://github.com/aptos-labs/aptos-core
```
Comment (teenu4) [inline developer-docs-site/docs/nodes/node-files.md:25]:
```suggestion
- **Git branch:** `mainnet` on https://github.com/aptos-labs/aptos-core
```

File: developer-docs-site/docs/nodes/node-files.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+- **Command to download:**
+  ```bash
+  wget -O validator.yaml https://raw.githubusercontent.com/aptos-labs/aptos-core/main/docker/compose/aptos-node/validator.yaml
```
Comment (teenu4) [inline developer-docs-site/docs/nodes/node-files.md:28]:
```suggestion
  wget -O validator.yaml https://raw.githubusercontent.com/aptos-labs/aptos-core/mainnet/docker/compose/aptos-node/validator.yaml
```

File: developer-docs-site/docs/nodes/node-files.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+:::tip Fullnode 
+Fullnode means either a validator fullnode or a public fullnode.
+:::
+
+- **Git repo:** `aptos-core`
+- **Git branch:** `mainnet` on https://github.com/aptos-labs/aptos-core
+- **Command to download:**
+  ```bash
+  wget -O docker-compose-fullnode.yaml https://raw.githubusercontent.com/aptos-labs/aptos-core/mainnet/docker/compose/aptos-node/docker-compose-fullnode.yaml
```
Comment (teenu4) [inline developer-docs-site/docs/nodes/node-files.md:77]:
```suggestion
  wget -O docker-compose.yaml https://raw.githubusercontent.com/aptos-labs/aptos-core/mainnet/docker/compose/aptos-node/docker-compose-fullnode.yaml
```

**Comment by rajkaramchedu** [comment]
@teenu4 Line 77 feels odd. Downloading a file and storing as a different filename. But not to worry. I will go over all these with our guys and get them right. We are not ready to merge this pr yet.

**Comment by teenu4** [comment]
> @teenu4 Line 77 feels odd. Downloading a file and storing as a different filename. But not to worry. I will go over all these with our guys and get them right. We are not ready to merge this pr yet.

Yes, I know, but it actually makes sense!
This way it's easier to use docker on a full node server. If we keep the original file name, we will need to specify custom `-f docker-compose-fullnode.yaml` every time we use `docker-compose` command.
I believe, in the repository, this filename is changed because they are in the same folder with validator's configuration file, but in practice, all validators rename it to default compose configuration file name (docker-compose.yaml).

**Review by gregnazario — APPROVED** [review]

**Pull request merged**